models.py

```diff
@@ -48,6 +94,11 @@ def forward(self, inputs: torch.Tensor, input_lengths: torch.Tensor):
       embedded_inputs = torch.cat((embedded_inputs, glove_embedded_inputs), dim=-1)
     if self.use_glove and not self.use_trainable_embeddings:
       embedded_inputs = self.glove(inputs)
+    if self.use_character_level_embeddings:
+      # compute character level embeddings
+      character_embedded_inputs = self.character_level_embeddings(character_inputs, character_inputs_lengths)
+      # concatenate embedded_inputs with character lever embeddings on the las dimention
```

Comment:
Maybe 'concatenate embedded_inputs with character LEVEL embeddings on the LAST DIMENSION' ?